src/components/Controllers/UserController.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { USER_RESOURCES } from '../../resources/Resources';
+
+class UserController {
+
+  static renderCreateForm({ newItem, handleChange }) {
+    const handleAdminChange = (e) => {
+      const value = e.target.value === 'true';
+      handleChange({ target: { name: USER_RESOURCES.ADMIN, value } });
+    };
+
+    return (
+      <>
+        <label>{USER_RESOURCES.NAME}:</label>
+        <input type="text" name={USER_RESOURCES.NAME} value={newItem[USER_RESOURCES.NAME]} onChange={handleChange} />
+        <label>{USER_RESOURCES.SURNAME}:</label>
+        <input type="text" name={USER_RESOURCES.SURNAME} value={newItem[USER_RESOURCES.SURNAME]} onChange={handleChange} />
+        <label>{USER_RESOURCES.EMAIL}:</label>
+        <input type="text" name={USER_RESOURCES.EMAIL} value={newItem[USER_RESOURCES.EMAIL]} onChange={handleChange} />
+        <label>{USER_RESOURCES.PASSWORD}:</label>
+        <input type="text" name={USER_RESOURCES.PASSWORD} value={newItem[USER_RESOURCES.PASSWORD]} onChange={handleChange} />
+        <label>{USER_RESOURCES.ADMIN}:</label>
+        <select name={USER_RESOURCES.ADMIN} value={newItem[USER_RESOURCES.ADMIN]} onChange={handleAdminChange}>
+        <option value={'false'}>False</option>
+        <option value={'true'}>True</option>
+        </select>
```

Comment:
for this area we should create a generic select component. that get props of defValue + onClick/onChange/ options and co....